sympy/utilities/lambdify.py

```diff
@@ -297,6 +297,20 @@ def lambdify(args, expr, modules=None, printer=None, use_imps=True,
     from sympy.core.symbol import Symbol
     from sympy.utilities.iterables import flatten
 
+    
+    print('+++++++++++++++')
+    print(type(expr))
+    print('---------------')
+    blacklist = [tuple, list, int, str, dict]
+    blacklisted = False
+    for x in blacklist:
+        if issubclass(type(expr), x):
+            blacklisted = True
+            break
```

Comment:
You can directly use isinstance for multiple objects. You don't need to iterate for each object.